src/lib/action.rs

```diff
@@ -780,7 +780,7 @@ mod test {
         let ins1 = Insert::new(n5, Some(NodeId::new(2)), 0);
         let ins2 = Insert::new(n6, Some(NodeId::new(2)), 1);
         let mov = Move::new(NodeId::new(6), NodeId::new(2), 0); // Swap "INT 100" and "INT 99".
-        // Change "+"" to "*".
+                                                                // Change "+"" to "*".
```

Comment:
Rustfmt has started to mess up these sorts of comments. The fix is to move the comment `// Swap ...` to above line 782, them rustfmt will put both comments in sensible places :(